examples/scenes/Cargo.toml

```diff
@@ -16,6 +16,7 @@ vello_svg = { path = "../../integrations/vello_svg" }
 anyhow = { workspace = true }
 clap = { workspace = true, features = ["derive"] }
 image = "0.24.5"
+instant = "0.1.12"
```

Comment:
According to `instant`'s documentation, one of the `stdweb` or `wasm-bindgen` features need to be updated.

I still don't have a working browser with WebGPU, so maybe this works? Perhaps `instant`'s docs need updating

Reply:
Good catch. I went ahead and enabled `wasm-bindgen` though AFAICT all the animated scenes still worked without it.